sdk/appconfiguration/azappconfiguration/policy_sync_token_parse_test.go

```diff
@@ -0,0 +1,223 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package internal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSyncTokenParseTokenEmpty(t *testing.T) {
+	_, err := parseToken("")
+	require.NotEmpty(t, err)
+}
+
+func TestSyncTokenParseTokenWithoutSeqNo(t *testing.T) {
+	{
+		tok1, err1 := parseToken("n=v")
+		require.Empty(t, err1)
+		require.NotEmpty(t, tok1)
+		require.Equal(t, tok1.id, "n")
+		require.Equal(t, tok1.value, "v")
+		require.Empty(t, tok1.seqNo)
+	}
+
+	{
+		tok2, err2 := parseToken("n=v;")
+		require.Empty(t, err2)
+		require.NotEmpty(t, tok2)
+		require.Equal(t, tok2.id, "n")
+		require.Equal(t, tok2.value, "v")
+		require.Empty(t, tok2.seqNo)
+	}
+}
+
+func TestSyncTokenParseTokenWithSeqNo(t *testing.T) {
+	{
+		tok1, err1 := parseToken("n=v;sn=1")
+		require.Empty(t, err1)
+		require.NotEmpty(t, tok1)
+		require.Equal(t, tok1.id, "n")
+		require.Equal(t, tok1.value, "v")
+		require.Equal(t, tok1.seqNo, 1)
+	}
+
+	{
```

Comment:
The blocks separated by `{` and `}` aren't necessary here, feels like this might be more a c/c++ thing.

Reply:
I'd say it is not very common in C/C++ as well, it's just me :) But I removed them.